src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/OpportunitiesService.cs

```diff
@@ -22,5 +23,23 @@ public async Task<List<OpportunityDto>> GetAllOpportunities()
 
             return JsonConvert.DeserializeObject<List<OpportunityDto>>(await response.Content.ReadAsStringAsync());
         }
+
+        public async Task<OpportunityDto> GetOpportunity(int id)
+        {
+            OpportunityDto opportunity = null;
+
+            var response = await _client.GetAsync($"pledges/{id}");
```

Comment:
I didn't spot this earlier, but the outer api is going to have to mirror, to some extent the UI. We need to introduce an `opportunities` controller into the outer api and get _that_ and not pledge. _It_, in turn will call the inner api's `pledges/{id}` route. In other words, opportunities should exist as a thing in the outer api, but not the inner.

We can deal with the other endpoint later as technical debt.

Reply:
Noting that this can be dealt with later as technical debt.